src/vmm/src/devices/virtio/queue.rs

```diff
@@ -899,21 +958,6 @@ mod verification {
         }
     }
 
-    mod stubs {
```

Comment:
this is a note to myself, but now that the functions under test are a lot simpler, I want to have a look at whether we can generalize the proofs in this module.

Reply:
wohoo, we could